src/core/core.cpp

```diff
@@ -342,6 +342,7 @@ void Core::start()
     else
     {
         qDebug() << "Self avatar not found";
```

Comment:
could you change this qdebug to "setting empty avatar"? so we don't print misleading debug messages.